PowerShellEditorServices.build.ps1

```diff
@@ -185,9 +189,12 @@ task LayoutModule -After Build {
     New-Item -Force $PSScriptRoot\module\PowerShellEditorServices\bin\Core -Type Directory | Out-Null
 
     Copy-Item -Force -Path $PSScriptRoot\src\PowerShellEditorServices.Host\bin\$Configuration\netstandard1.6\* -Filter Microsoft.PowerShell.EditorServices*.dll -Destination $PSScriptRoot\module\PowerShellEditorServices\bin\Core\
+    Copy-Item -Force -Path $PSScriptRoot\src\PowerShellEditorServices.Host\bin\$Configuration\netstandard1.6\UnixConsoleEcho.dll -Destination $PSScriptRoot\module\PowerShellEditorServices\bin\Core\
```

Comment:
@tylerl0706 before you can ship these binaries with PSES and the PowerShell extension, you'll need to request OSS approval for [Patrick's library](https://github.com/SeeminglyScience/UnixConsoleEcho).  You may also need to include his copyright in a Third Party Notices file like the one in the PS extension.  I highly doubt they will reject the request to use this code but better to follow the necessary process anyway :)